Argo.xcodeproj/project.pbxproj

```diff
@@ -121,6 +133,15 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		D03579C61B2B7A14005D26AE /* CopyFiles */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
```

Comment:
actually nvm ... I'm assuming this was created by default when the watchOS target was created. If so, then just leave it.